mobile/src/main/res/values/strings.xml

```diff
@@ -54,9 +54,9 @@
     <string name="settings_clear_default_sitemap">Clear default sitemap</string>
     <string name="settings_no_default_sitemap">No default sitemap selected</string>
     <string name="settings_current_default_sitemap">Current default sitemap: %1$s</string>
-    <string name="settings_openhab_sslsettings">Encryption settings</string>
-    <string name="settings_openhab_alt_connection">Remote server settings</string>
-    <string name="settings_openhab_connection">Local server settings</string>
+    <string name="settings_openhab_sslsettings">Encryption</string>
+    <string name="settings_openhab_alt_connection">Remote</string>
+    <string name="settings_openhab_connection">Local</string>
```

Comment:
`Local server` / `Remote server`? Only local/remote seens a bit short. I saw the previous discussion, but I think omitting both 'connection' and 'settings' oversimplifies things and makes the string less clear for no real gain.

Reply:
Your suggestion is a problem on its own. The menu entry is not necessarily related to a "server". Imagine your wife using the app or a user who's landlord provides an openHAB system for the central heating system. Those users do not know the server, they want to connect to a service using address and credentials.

All: I've been thinking about this for a while: Are you considering support of multiple accounts? Part of the redesign needed for such a function could also be the redesign of this local/remote part. 

My proposal would be to step away from local and remote completely. Instead use something like:

![grafik](https://user-images.githubusercontent.com/2870104/34826889-9fb21b2a-f6d8-11e7-8ae4-8d067ec150fc.png)

https://ninjamock.com/s/6DQ7MWx

Reply:
> Are you considering support of multiple accounts?

In a very very very long term: I think so, yes. See #10 for the discussion :)

I'm not sure, if everyone can easily know when a "primary connection" and when the "alternative connection" will be used. One could think, that the primary connection is always used, and only, if it is not reachable, the alternative connection is used. This is IHMO false, as the primary would not be used when using mobile data (it would not even be tried to connect to this service) :/

However, I don't have any better idea, so :(

Reply:
We should also look at #10 and how the GUI needs to be changed for that.
The heading of "Local" and "Remote" is "Connection", so we would have it twice.

Reply:
IMHO there is nothing wrong with a tiny bit of redundancy, as long as it isn't obviously repetitive.

Reply:
I don't want to continue this discussion here but just one more thought: How likely is it that a user needs both URLs? If a system is accessible via a public address, why not use it all the time. There is no technical reason why anyone would want to provide the local address in this case.